docs/main/docs/components/sfx-overview.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 1
+---
+
+# What is a Sideeffect
+
+A Sideeffect is the description of a transaction that needs to be executed on an external consensus system. When initialised, the creator of the Sideeffect (can be a user or smart contract) sets all the neccessary attribtues and commits it to the Circuit, where Executors are able to lacate them.
+
+
+## Sideeffect structure
+```
+pub struct SideEffect<AccountId, BalanceOf> {
+    pub target: TargetId,
+    pub max_reward: BalanceOf,
+    pub insurance: BalanceOf,
+    pub encoded_action: Bytes,
+    pub encoded_args: Vec<Bytes>,
+    pub signature: Bytes,
+    pub enforce_executor: Option<AccountId>,
+}
+```
+
+#### target:
+`target` describes the destination consensus system the Sideeffect should be executed on. 
+
+#### max_reward 
+`max_reward` sets the reward for the Executor in `TRN`.
+
+#### insurance
+`insurance` sets the minimum amount of insurance and Executor has to Bond. This is important for optimistic transactions.
```

Comment:
insurance is just as important in escrow. It incentives executors to confirm the transactions they have one the bidding on